ksql-engine/src/main/java/io/confluent/ksql/physical/AddTimestampColumn.java

```diff
@@ -36,6 +36,7 @@ public void init(ProcessorContext processorContext) {
       public GenericRow transform(GenericRow row) {
         if (row != null) {
           row.getColumns().add(0, processorContext.timestamp());
+          return row;
```

Comment:
Is this intentional? Seems pretty pointless to add this.

Reply:
Good catch - I moved some of the code around so I could set a breakpoint for the null case. I'll revert.